Commons/src/main/java/eu/neclab/ngsildbroker/commons/tools/HttpUtils.java

```diff
@@ -678,7 +716,7 @@ public static RestResponse<String> generateNotification(ArrayListMultimap<String
 			acceptHeader.add("application/json");
 		}
 		String body = getReplyBody(acceptHeader, AppConstants.QUERY_ENDPOINT, headers, notificationData, true,
-				ldContext, context, geometryProperty).await().indefinitely();
+				ldContext, context, geometryProperty, "").await().indefinitely();
```

Comment:
Please use null instead of empty string.

Reply:
done